oneview_redfish_toolkit/tests/api/test_computer_system.py

```diff
@@ -85,14 +84,3 @@ def test_serialize(self):
             self.fail("Failed to serialize. Error: ".format(e))
 
         self.assertEqual(self.computer_system_mockup, result)
-
-    def test_get_oneview_power_configuration(self):
```

Comment:
You has removed this test because you has moved the method tested to `api/util/power_option.py` file , right?
But I didn't see a test to the method of `power_option.py`, do you think that is good mainten it?

Reply:
Yes, it is. I create a test_power_option test cases on api. Please check them.